pydantic/_internal/_decorators.py

```diff
@@ -239,8 +325,9 @@ def build(model_dc: type[Any]) -> DecoratorInfos:  # noqa: C901 (ignore complexi
 
         # reminder: dicts are ordered and replacement does not alter the order
         res = DecoratorInfos()
-        for base in model_dc.__bases__[::-1]:
-            existing = cast(Union[DecoratorInfos, None], getattr(base, '__pydantic_decorators__', None))
+        # use __orig_bases__ to support TypedDict but fall back to __bases__ for models and dataclasses
```

Comment:
Are you sure this comment belongs here? I don't see how it applies to the code below.

Reply:
I agree it can be removed now, thanks.